docs/tasks/maintenance/import_controller_upgrade.md

```diff
@@ -0,0 +1,96 @@
+---
+sidebar_position: 2
+---
+
+# Upgrade Turtles import controller
+
+## Context
+
+When Turtles imports a CAPI cluster into Rancher, a number of Kubernetes resources must be created to represent the CAPI cluster in Rancher. The main resource is the one representing the CAPI cluster as a Rancher cluster, so it can be viewed and managed via Rancher:
+```
+clusters.provisioning.cattle.io
+```
+We also call this resource a `v1` cluster. If you were using Turtles before v0.9.0, the default import controller was based on generating this resource for every CAPI cluster that is imported into Rancher. The controller we will be migrating to was disabled behind a feature gate `managementv3_cluster`.
+
+With the new Turtles release, the `v3` import controller is no longer behind a feature gate and will be the default import logic. This means we are moving to a different strategy for importing clusters, and switching to creating a different type of Rancher cluster resource:
```

Comment:
If we are already sure, we can highlight it clearly from which release and onwards